test/unit/riscv/InstructionTest.cc

```diff
@@ -0,0 +1,587 @@
+#include "../ConfigInit.hh"
```

Comment:
Most comments made in the equivalent aarch64 file apply to this file as well

Reply:
Comments fixed / added